packages/cf-component-copyable-textarea/package.json

```diff
@@ -15,6 +15,7 @@
   },
   "devDependencies": {
     "assert-equal-jsx": "^1.0.3",
+    "cf-util-text": "^2.0.0",
```

Comment:
The latest version doesn't use the gettext settext stuff in cf-util-text. There no dependency on it anymore. How does it break your build?